routes/web.php

```diff
@@ -14,39 +14,39 @@
 |
 */
 
-Route::get('/', 'MatchdayController@index')->middleware('auth');
-Route::get('/alletermine', 'MatchdayController@index')->middleware('auth');
+Route::get('/presentation', function () {
+    return view('presentation');
+});
 
-Route::get('/termine/bearbeiten', function () {
-    return view('matchdays.edit');
-})->middleware('auth');
-// Route::get('/termine/bearbeiten', 'MatchdayController@edit')->middleware('auth');
 
-Route::get('/termine/erstellen', 'MatchdayController@create')->middleware('auth');
+Route::group(['middleware' => 'loggedin'], function () {
```

Comment:
See my comment in app/Http/Kernel.php
```suggestion
Route::group(['middleware' => 'auth'], function () {
```